flair/datasets/sequence_labeling.py

```diff
@@ -732,6 +732,64 @@ def __offset_docstarts(file_in: Union[str, Path], file_out: Union[str, Path]):
                 if line.startswith('-DOCSTART-'):
                     f.write("\n")
 
+class PERSON_NER(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "ner",
+            in_memory: bool = True,
+            **corpusargs,
+    ):
+        """
+        Initialize the PERSON_NER corpus for person names. The first time you call this constructor it will automatically
+        download the dataset.
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param tag_to_bioes: NER by default, need not be changed, but you could also select 'pos' to predict
+        POS tags instead
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+        # column format
+        columns = {0: "text", 1: "ner"}
+
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = flair.cache_root / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        conll_path = "https://github.com/das-sudeshna/genid"
+
+        # download files if not present locally
+        cached_path(f"{conll_path}.conll", Path("datasets") / dataset_name)
```

Comment:
You need to download all four files and concatenate them into one before instantiating the corpus

Reply:
Added the download of all the files and the concatenating of them.